app/code/Magento/Catalog/view/adminhtml/web/js/category-tree.js

```diff
@@ -88,9 +136,10 @@ define([
             if (!node) {
                 return result;
             }
+
             result = {
                 data: {
-                    title: node.name + ' (' + node['product_count'] + ')'
+                    title: decodeEntities(node.name) + ' (' + node['product_count'] + ')'
```

Comment:
Instead of implementing custom function use `_.unescape`

Reply:
I have tried using the `_.unesacpe` function but that didn't work out with the above-escaped string.